lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -336,6 +350,86 @@ class CategoryRenderer {
   }
 
   /**
+   * Render a screenshot filmstrip and metrics bars
+   * @param {!Array<!ReportRenderer.AuditJSON>} metricAudits
+   * @param {*} artifacts
+   * @return {?Element}
+   */
+  _renderPerfTimeline(metricAudits, artifacts) {
+    if (!artifacts)
+      return;
+    const tracePass = artifacts['traces'] ? artifacts['traces']['defaultPass'] : null;
+    if (!tracePass)
+      return;
+
+    const fmp = metricAudits.find(a => a.id === 'first-meaningful-paint').result;
+    if (!fmp || !fmp['extendedInfo'])
+      return;
+
+    const tti = metricAudits.find(a => a.id === 'time-to-interactive').result;
+    if (!tti || !tti['extendedInfo'])
+      return;
+
+    const navStart = fmp['extendedInfo']['value']['timestamps']['navStart'];
+    const markers = [
+      {
+        title: 'First contentful paint',
```

Comment:
question:  should we include the audit entry for these metrics if we have them here?
probably not
but that means we'll have to include their helpText in their visualization.

![image](https://cloud.githubusercontent.com/assets/39191/25791313/4da06be8-3374-11e7-86e3-26b983b42321.png)


if we do that, then we'll only show bars for the metrics audits we have, with the exception of EIL, which is our non pageload metric. 
and this also means if we want to show "first contentful paint" or "first visual change" etc.. then they should be real top-level audits.

that sound reasonable?